src/sql.rs

```diff
@@ -272,7 +269,7 @@ impl Sql {
             ) -> anyhow::Result<H>,
     {
         let lock = self.pool.read().await;
-        let pool = lock.as_ref().ok_or(Error::SqlNoConnection)?;
+        let pool = lock.as_ref().ok_or(format_err!("No SQL connection"))?;
```

Comment:
These clippy errors are interesting.  I guess it indicates that using an enum with an Display and Error impl is faster to return than an anyhow error.  Not that I think this matters, I just never thought about this aspect before.